core/field.js

```diff
@@ -202,6 +202,13 @@ Blockly.Field.prototype.isDirty_ = true;
  */
 Blockly.Field.prototype.visible_ = true;
 
+/**
+ * Can the field value be changed using the editor on an editable block?
```

Comment:
Nit: For consistency, "Whether the field value can be changed using the editor on an editable block."

Reply:
Sounds good, I made the change.  I'm holding off on making another commit before the serialization questions are resolved.

Reply:
Actually one follow-up: Should other comments be changed as well?  I used a question here because some other properties were commented that way.

Reply:
You're right, this is consistent with how this file does it. I wasn't familiar with this style inside of Blockly before. You can just leave it like you had it. Sorry for the churn.